.prettierignore

```diff
@@ -6,4 +6,4 @@
 apps/*/build
 libs/*/build
 libs/*/@types
-libs/ui-lib-tests/cypress/fixtures/static-ip/files/test.yaml
+libs/ui-lib-tests/cypress/fixtures/*/files
```

Comment:
why do you need this?

Reply:
because prettier try to format yaml files and give an error. With this change, prettier ignores yaml files

Reply:
Do we need it to format yaml/yml files?

Reply:
I think not. @jgyselov add this rule for static ips and I made more global for all yaml files in tests.

Reply:
(We specifically need it not to format yaml files. Or any files that we use to test file upload components.)